free-programming-books-id.md

```diff
@@ -66,7 +66,11 @@
 * [Database PostgreSQL, Pemrograman Python, dan SMS Gateway](http://rab.co.id/files/python/bukupython2.pdf.gz) - (PDF)
 * [Workshop Python 101](http://sakti.github.io/python101/)
 
+* [Dasar Pemrograman Python](https://github.com/purwnt/Belajarpython.git)
```

Comment:
Good resource, but check travis-ci to see what the linter is failing for. (I have lots of Hacktoberfest submissions to process!) Better to  link to the page for the repo rather than the .git file

Reply:
do not use the .git link